packages/addons/service/snapclient/source/resources/language/English/strings.po

```diff
@@ -0,0 +1,68 @@
+# Kodi Media Center language file
+# Addon Name: snapclient
+msgid ""
+msgstr ""
+
+msgctxt "#30000"
+msgid "Snapclient"
+msgstr ""
+
+msgctxt "#30001"
+msgid "List sound cards"
+msgstr ""
+
+msgctxt "#30002"
+msgid "Sound card"
+msgstr ""
+
+msgctxt "#30003"
+msgid "Host ID"
+msgstr ""
+
+msgctxt "#30004"
+msgid "Port"
+msgstr ""
+
+msgctxt "#30005"
+msgid "Priority"
+msgstr ""
+
+msgctxt "#30006"
+msgid "Latency"
+msgstr ""
+
+msgctxt "#30007"
+msgid "Kodi"
+msgstr ""
+
+msgctxt "#30008"
+msgid "Stop Snapclient when Kodi plays"
+msgstr ""
+
+msgctxt "#30009"
+msgid "Rasperry Pi"
+msgstr ""
+
+msgctxt "#30010"
+msgid "Manage onboard audio"
+msgstr ""
+
+msgctxt "#30011"
+msgid "Onboard audio playback route"
+msgstr ""
+
+msgctxt "#30012"
+msgid "auto detect"
```

Comment:
`msgid "Autodetect"` ? (idk where it is shown)

Reply:
Same as in Librespot

Reply:
https://en.wiktionary.org/wiki/autodetect so i guess `Autodetect` should be okay